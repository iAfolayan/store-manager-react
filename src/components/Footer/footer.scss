.footer {
  .ui.inverted.pointing.secondary.menu {
    justify-content: center;
  }
  margin: 0;
}
