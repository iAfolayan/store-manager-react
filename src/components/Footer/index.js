import React from 'react';
import PropTypes from 'prop-types';
import './Footer.scss';

const Footer = ({ footerClassName }) => (
  <div className={footerClassName}>
    <span>@ 2018 - Andela Developer Challenge</span>
  </div>
);

Footer.prototype = {
  footerClassName: PropTypes.string.isRequired
}

export default Footer;
