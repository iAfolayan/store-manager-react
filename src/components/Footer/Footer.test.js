import React from 'react';
import { shallow } from 'enzyme';
import Footer from '.';

describe('Test Footer Component', () => {
  test('should matches the snapshot', () => {
    const wrapper = shallow(<Footer footerClassName="" />);
    expect(wrapper).toMatchSnapshot();
  });
});
