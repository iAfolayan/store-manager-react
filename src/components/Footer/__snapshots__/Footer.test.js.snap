// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Footer Component should matches the snapshot 1`] = `
<div
  className=""
>
  <span>
    @ 2018 - Andela Developer Challenge
  </span>
</div>
`;
